resources/s3_error_messages.json

```diff
@@ -194,5 +194,9 @@
   "MissingContentLength": {
     "Description": "You must provide the Content-Length HTTP header.",
     "httpcode": 411
+  },
+  "IllegalVersioningConfigurationException": {
```

Comment:
This error should be removed and replaced with whatever AWS returns, which I believe is MalformedXML?

Reply:
done